### ISSUE TITLE
[BUG] fix `releases.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/OpenSource-API#35]
### Description

Update the 2 actions to the latest version

```diff
- uses: actions/checkout@v2
+ uses: actions/checkout@v3
```

```diff
- uses: TriPSs/conventional-changelog-action@v3.7.1
+ uses: TriPSs/conventional-changelog-action@v3
```

File path - https://github.com/Pradumnasaraf/OpenSource-API/blob/main/.github/workflows/releases.yml

**Comment by tarunsamanta2k20** [comment]
I would like to work on this issue

**Comment by Harsh-Daga** [comment]
Heyy @Pradumnasaraf ,
If no one is working on this issue, I would like to contribute to this.

**Comment by Pradumnasaraf** [comment]
> Heyy @Pradumnasaraf , If no one is working on this issue, I would like to contribute to this.

Yes, go ahead

**Comment by AyonChowdhury1307** [comment]
If no one is working on this issue, I would like to contribute to this

**Comment by AyonChowdhury1307** [comment]
i want to contribute

**Comment by Pradumnasaraf** [comment]
> i want to contribute

Hey, please at the issue which is not allotted to people. If you comment on an issue on which people are already working it will create extra notifications.

**Comment by Pradumnasaraf** [comment]
fixed in #46
